### PR TITLE
fix error and update

### PR DESCRIPTION
**Pull request by ScolderCreations**

### CODE REVIEW

File: spellcheck.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
   "musci": "music",
   "musif": "music",
   "musik": "music",
-  "muzeum": "museum"
```
Comment (hello-smile6) [inline spellcheck.json:10]: Good catch!

File: spellcheck.json [hunk 2]
```diff
@@ -7,6 +7,10 @@
...
-  "jewlry": "jewelry"
+  "muzeum": "museum",
+  "jewlry": "jewelry",
+  "clik": "click",
+  "clikc": "click",
+  "cick": "kick",
```
Comment (hello-smile6) [inline spellcheck.json:14]:
```suggestion
```
That could also be `click`.

**Review by hello-smile6 — CHANGES_REQUESTED** [review]
Nice work! Just worried about spellcheck.json#14

File: spellcheck.json (hunk 2 above)
Comment (ScolderCreations) [reply spellcheck.json:14]: Could be. Another reason why there should be array support…

File: spellcheck.json [hunk 3]
```diff
@@ -11,6 +11,5 @@
   "jewlry": "jewelry",
   "clik": "click",
   "clikc": "click",
```
Comment (hello-smile6) [inline spellcheck.json:13]: Reorder? Otherwise, the user's code would get broken if they had line 13's entry somewhere.

**Review by hello-smile6 — CHANGES_REQUESTED** [review]
Looks good, just that one change.

File: spellcheck.json (hunk 2 above)
Comment (hello-smile6) [reply spellcheck.json]: @ScolderCreations That'd require an update to the codebase.

File: spellcheck.json (hunk 3 above)
Comment (ScolderCreations) [reply spellcheck.json:13]: Yea, that makes sense.

**Review by hello-smile6 — APPROVED** [review]
Looks good! Unless someone intentionally makes a file that's invalid, this should work!

**Pull request merged**